crates/rolldown/src/stages/generate_stage/code_splitting.rs

```diff
@@ -22,14 +22,14 @@ impl<'a> GenerateStage<'a> {
     };
     let meta = &self.link_output.metas[module_id];
 
-    if !module.is_included {
-      return;
-    }
-
-    if module_to_bits[module_id].has_bit(entry_index) {
-      return;
+    // Here if the module only has re-export statements, the `is_included` is false at `module_side_effects: false`.
+    // But we also need to include it's dependencies.
+    if module.is_included {
+      if module_to_bits[module_id].has_bit(entry_index) {
+        return;
+      }
+      module_to_bits[module_id].set_bit(entry_index);
```

Comment:
Because the `report-star.js` is_included is false, so here need to determine it's dependencies. If we could be change is_included to be true, we don't need this, i'm not sure for it. @IWANABETHATGUY

Reply:
https://github.com/rolldown/rolldown/blob/9719b2a61a0c43ef6c4f0aa08cef12935ab2b15d/crates/rolldown/src/stages/link_stage/tree_shaking.rs#L84-L100

Related code.

Reply:
Where does it panic before this PR? I feel we shouldn't included this module but we should also not panic.

Reply:
See crates/rolldown/src/stages/generate_stage/code_splitting.rs:136:7

Reply:
Get it. I'll consider this as a workaround fix. This is more of a bug of linking.